docs/source/diagnostics-single.rst

```diff
@@ -1,54 +1,29 @@
-Diagnostics
-===========
+Diagnostics (single)
```

Comment:
Perhaps `Diagnostics (local)` or `Diagnostics (single machine)`? Current title sounds off.

Reply:
Agreed. Thanks for the suggestion. I think I've hit all of the various links.